### PR TITLE
Tests to check website quality

### PR DESCRIPTION
**Pull request by kwhitehall**

### CODE REVIEW

File: tests/htmlPageChecker.py [hunk 1]
```diff
@@ -0,0 +1,238 @@
+from selenium import webdriver
```
Comment (chrismattmann) [inline tests/htmlPageChecker.py:17]: @kwhitehall we should put an ALv2 license on the header for this.

**Comment by chrismattmann** [comment]
@kwhitehall I had a thought too, once we put the ALv2 header on this - we could also make the tests run via travis-ci no?

**Comment by kwhitehall** [comment]
@chrismattmann license added. I would have to look into travis-ci further. First pass, it looks possible, but I am not sure how to implement ... yet.

**Comment by chrismattmann** [comment]
LGTM thanks @kwhitehall !

**Pull request merged**